Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -92,6 +92,22 @@ void ShutdownSoundStream()
   INFO_LOG(AUDIO, "Done shutting down sound stream");
 }
 
+std::string GetDefaultSoundBackend()
+{
+  std::string backend = BACKEND_NULLSOUND;
+#if defined __linux__
+  if (AlsaSound::isValid())
```

Comment:
I like the separation of concerns. Would also be a good place to fallback to PulseAudio/whatever else.